### ISSUE TITLE
Elementor stucked while loading

### ISSUE THREAD

**Issue by lukaszpul** [pojome/elementor#2537]
Elementor stuck on the loading page.
I've updated Elementor today and cant it load now.
I followed all troubleshooting steps already but it did not help.
1. Deactivated all plugins,
2. Increased memory to 128MB
3. Changed theme to default

System info

== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.23 (Win64) PHP/5.6.25
	MySQL version: 5.7.14
	PHP Version: 5.6.25
	PHP Max Input Vars: 2500
	PHP Max Post Size: 8M
	GD Installed: Yes
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.8.2
	Site URL: http://localhost/wordpress
	Home URL: http://localhost/wordpress
	WP Multisite: Yes
	Max Upload Size: 1 MB
	Memory limit: 128M
	Permalink Structure: /blog/%year%/%monthnum%/%day%/%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: radziewicz.lukasz@gmail.com
	Debug Mode: Active

== Theme ==
	Name: OceanWP
	Version: 1.3.9
	Author: Nick
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:56.0) Gecko/20100101 Firefox/56.0

== Active Plugins ==
	Image slider with description
		Version: 8.1
		Author: Gopi Ramasamy

	Ocean Extra
		Version: 1.3.2
		Author: OceanWP

== Network Plugins ==
	Elementor
		Version: 1.7.8
		Author: Elementor.com

	Elementor Addon Elements
		Version: 0.5
		Author: Webtechstreet.com

	Ocean Extra
		Version: 1.3.2
		Author: OceanWP

== Debug ==
	Errors: 
		NoTemplateError: Could not find template: "#tmpl-elementor-repeater-row"
			at http://localhost/wordpress/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5 - 22:17423
			2017-10-03 13:45 +00:00
			x 3 times

		TypeError: elementor.channels.panelElements.request(...) is undefined
			at http://localhost/wordpress/wp-content/plugins/elementor/assets/js/editor.min.js?ver=1.7.6 - 2:108550
			2017-10-01 11:18 +00:00
			x 371 times

		TypeError: elementor.getPanelView(...).getCurrentPageView(...).showView is not a function
			at http://localhost/wordpress/wp-content/plugins/elementor/assets/js/editor.min.js?ver=1.7.4 - 2:50130
			2017-09-19 20:47 +00:00
[system-info-localhost-03-10-2017-1.txt](https://github.com/pojome/elementor/files/1352567/system-info-localhost-03-10-2017-1.txt)

**Comment by norewp** [comment]
Please try the latest version and confirm

**Comment by lukaszpul** [comment]
Looks like their update wasn't most fortunate. It works fine after another update. Thanks

**Comment by Klemart3D** [comment]
I have same error but only in browsers Console (not in Elementor debugger), with latest version of Elementor (v1.8.0) + fresh install of Wordpress 4.8.3 with default "Twenty Seventeen" theme.

Screenshots with Chrome and Firefox (and same with Opera and Safari) :

![](https://img4.hostingpics.net/pics/98009263ch.jpg)

![](https://img4.hostingpics.net/pics/96799581ff.jpg)

```					
== Server Environment ==
	Operating System: Darwin
	Software: Apache
	MySQL version: 5.6.35
	PHP Version: 5.6.31
	PHP Max Input Vars: 1000
	PHP Max Post Size: 32M
	GD Installed: Yes
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.8.3
	Site URL: http://monsite.fr
	Home URL: http://monsite.fr
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 40M
	Permalink Structure: /%postname%/
	Language: fr-FR
	Timezone: Europe/Paris
	Debug Mode: Active

== Theme ==
	Name: Twenty Seventeen
	Version: 1.3
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: fr_FR
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:56.0) Gecko/20100101 Firefox/56.0

== Active Plugins ==
	Elementor
		Version: 1.8.0
		Author: Elementor.com

== Debug ==
	Errors: There are no errors to display
```

**Comment by regenrek** [comment]
Can confirm this bug on 1.8.0. I reverted back to 1.7.12 and it works again.

**Comment by bainternet** [comment]
Thanks for reporting, should be fixed in 1.8.1

**Comment by berkinimamoglu** [comment]
https://www.youtube.com/watch?v=oEZlWdO38SQ this should solve this.
